hack/manifest-gen/assets/charts/eck/values.yaml

```diff
@@ -0,0 +1,92 @@
+# Core configuration values
+operator:
+  # Name of the operator installation.
+  name: elastic-operator
+  # Namespace to install the operator to.
+  namespace: elastic-system
+  # Version of the operator. Should be set to the desired container image tag.
+  version: 1.2.0
+
+  # Labels to be added to the operator. Useful if multiple operators are to be deployed to the cluster.
+  # These labels are used by the webhook service if it is enabled as well.
+  selectorLabels:
+    control-plane: elastic-operator
+
+  # Container image options.
+  image:
+    # Full path to the operator container image excluding the tag. The tag value is derived from the operator.version field.
+    repository: docker.elastic.co/eck/eck-operator
```

Comment:
`image.repository` sounds like the docker registry to me.
How about:
```yaml
image: docker.elastic.co/eck/eck-operator
imagePullPolicy: IfNotPresent
```

I can imagine people search for `imaagePullPolicy` directly since it's the k8s name already.

Reply:
It's normal Helm convention.